requirements.txt

```diff
@@ -7,4 +7,5 @@ requests==2.22.0
 semantic-version==2.8.2
 setuptools
 six
+tqdm
```

Comment:
The `requirements.txt` file tracks `bench`'s dependencies, not the `install.py` script's.

Remove it from here. What you want to do is add `import_with_install('tqdm')` in the `setup_script_requirements` function which manages the script's dependencies.

https://github.com/frappe/bench/blob/f3809b00acc4bfa586e9a12116fb6bd262d3226e/install.py#L418-L421

Reply:
Thank you, have made the changes. :)